shared/schemas/workflow-options-output-schema.json

```diff
@@ -0,0 +1,37 @@
+{
+    "$schema": "http://json-schema.org/draft-07/schema#",
+    "title": "Workflow Results",
+    "description": "Results of a workflow execution",
+    "type": "object",
+    "properties": {
+        "completedWith": {
+            "description": "The state of workflow completion.",
+            "type": "string",
+            "enum": [
+                "error",
+                "success"
+            ]
+        },
+        "description": {
+            "description": "High-level summary of the current status, free-form text, human readable.",
+            "type": "string"
+        },
+        "nextWorkflows": {
+            "description": "List of workflows suggested to run next. Items at lower indexes are of higher priority.",
+            "type": "array",
+            "items": {
+                "$ref": "workflow-ref-schema.json"
+            }
+        },
+        "options": {
+            "description": "Additional structured output of workflow processing. This can contain identifiers of created resources, links to resources, logs or other output.",
+            "type": "array",
+            "items": {
+                "$ref": "workflow-option-schema.json"
```

Comment:
let's rename this file name according to the name that will replace the `options` property above.